library/src/main/java/bio/terra/landingzone/db/LandingZoneDao.java

```diff
@@ -65,8 +66,8 @@ public LandingZoneDao(LandingZoneDatabaseConfiguration landingZoneDatabaseConfig
       transactionManager = "tlzTransactionManager")
   public UUID createLandingZone(LandingZoneRecord landingzone) {
     final String sql =
-        "INSERT INTO landingzone (landingzone_id, resource_group, subscription_id, tenant_id, billing_profile_id, created_date, definition_id, definition_version_id, display_name, description, properties) "
-            + "values (:landingzone_id, :resource_group, :subscription_id, :tenant_id, :billing_profile_id, :created_date, :definition_id, :definition_version_id, :display_name, :description,"
+        "INSERT INTO landingzone (landingzone_id, resource_group, subscription_id, tenant_id, billing_profile_id, region, created_date, definition_id, definition_version_id, display_name, description, properties) "
```

Comment:
I'm confused, this seems to be persisting the region even though the PR description says that's not in scope.

Reply:
It's just the logic in the flight that I'm excluding, since that's where it got complicated.

It's always going to be `null` for now, because nothing is setting the field anywhere.